src/manipulation.js

```diff
@@ -17,7 +17,7 @@ define([
 
 var
 	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,
-	rtagName = /<([\w:]+)/,
+	rtagName = /<([\w:-]+)/,
```

Comment:
So we have ":" in manipulation but not in core, also no tests for ":".

@dmethvin how you want to handle this? We either could add tests for ":" names, which probably means we would need to support and add tests for the core module too.

Or just remove this sign.

/cc @timmywil